Library/Formula/nanomsgxx.rb

```diff
@@ -0,0 +1,42 @@
+class Nanomsgxx < Formula
+  desc "Nanomsg binding for C++11"
+  homepage "https://achille-roussel.github.io/nanomsgxx/doc/nanomsgxx.7.html"
+  url "https://github.com/achille-roussel/nanomsgxx/archive/0.1.tar.gz"
+  sha256 "50f6d2507f264f048e0234e78bbbfe80004d636d990fefc4e1f53a5434c4aa63"
+
+  option "with-debug", "Compile with debug symbols"
+
+  depends_on "pkg-config" => :build
+  depends_on :python
```

Comment:
Should be `depends_on :python if MacOS.version <= :snow_leopard` I suspect. Actually needed for run or only build?